.travis.yml

```diff
@@ -37,3 +37,9 @@ matrix:
   allow_failures:
    - env: GHCVER=head
   fast_finish: true
+
+notifications:
+  irc:
+    channels:
+      - "chat.freenode.net##haskell-cabal"
```

Comment:
Ugh, it looks like I'll have to set up some autoinviter for Slack, since "public Slack" is not a by default supported operation. Let's leave it for now, I guess.